### ISSUE TITLE
Unable to connect with expired IDE

### ISSUE THREAD

**Issue by ericpaulsen** [coder/jetbrains-coder#144]
when i first try to connect to a workspace using the gateway plugin, I'm returned this error:

```
Could not get host jstack: 

Details:
An error occurred while executing command: 'get-jstack --ide-path=/home/coder/.cache/JetBrains/RemoteDev/dist/36e40bdbd952a_ideaIU-2022.2.4 --project-path=/home/coder'
Exit code: 1
```

upon clicking `Retry`, the connection is successfully established.

**Comment by matifali** [comment]
JetBrains Gateway expires the IDE backends, at least for EAP versions, after a specific duration, and that causes this error.
related discussions: https://intellij-support.jetbrains.com/hc/en-us/community/posts/10898451233682-Remote-development-SSH-connection

**Comment by matifali** [comment]
We have added a flag `latest=true` since version `v1.0.28` that always downloads the latest IDE backend.
https://registry.coder.com/modules/jetbrains-gateway

**Comment by matifali** [comment]
We have started working on the ToolBox integration and going forward I don't think gateway will receive much support. 

I will bring this up with gateway team if it comes from any users.

**Comment by code-asher** [comment]
Yup that is fair.  Hopefully Toolbox handles expired IDEs more gracefully!
